aws/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -135,6 +109,11 @@ storage:
             --net=host \
             --dns=host \
             --exec=/kubectl -- --kubeconfig=/etc/kubernetes/kubeconfig delete node $(hostname)
+    - path: /etc/kubernetes/kubelet-wrapper
+      filesystem: root
+      mode: 0744
+      contents:
+        inline: ${kubelet_wrapper_inline}
```

Comment:
Just curious: why put it inline and be so close to the user_data limit? Would it make sense to copy the file it with a null resource in terraform? (as we do for secrets, see the copy_controllers_secrets, for example)

Reply:
@rata if we copy with null resource, then it won't work for autoscaled nodes.

Reply:
Oh, good point. This is for the workers :man_facepalming: Thanks!

Reply:
@johananl mentioned that ssh with a null resource would not work with auto-scaling. It works for secrets because they get copied to the first controller node only. But the wrapper scripts need to be available on new nodes as well.

Reply:
Yes, sorry, my bad :)

Reply:
I think we should be moving away from dynamic provisioning over SSH. The more we do this, the more fragile our installer is going to be. Instead, we should try to rely on cloud-native mechanisms as much as possible (e.g. cloud-init user data).

Reply:
I totally agree, :100: 

Just that being so close to the limit of user_data seems dangerous. Therefore, I was wondering what evil was worst (until we find a reliable fix for the user data size) :-D

But, in this case, it seems there is no other option so nothing to argue other than we might want to reduce this script size as much as possible :)